VideoLocker/res/values/styles.xml

```diff
@@ -98,4 +98,8 @@
         <item name="android:textSize">12sp</item>
     </style>
 
+    <style name="AppTheme.Dark" parent="@android:style/Theme.Holo.Light">
+        <item name="android:actionBarItemBackground">@color/blue_bar</item>
```

Comment:
Or maybe just a name to match this http://ux.edx.org/elements/colors/ looks like it's primary-accent?